karma-src.conf.js

```diff
@@ -46,7 +46,7 @@ module.exports = function(config) {
       'karma-jquery',
       'karma-mocha',
       'karma-ng-html2js-preprocessor',
-      'karma-phantomjs-launcher',
+      'karma-chrome-launcher',
```

Comment:
This is for testing, right? Why this change?

Reply:
PhantomJS is not supported anymore: ariya/phantomjs#15344